src/frequenz/client/dispatch/__main__.py

```diff
@@ -52,8 +176,15 @@
     show_envvar=True,
     required=True,
 )
+@click.option(
+    "--raw",
```

Comment:
Possible future improvements:
* Split `--raw` from `--no-color`
* Set color on/off automatically based on `sys.stdin.isatty()`

Reply:
> Set color on/off automatically based on sys.stdin.isatty()

that actually seems to happen automatically, at least there is no color if I pipe it through cat or forward it into a file

Reply:
Ah cool. Extra point for `click` 💯